### PR TITLE
🌍 Arabic Language added

### PR DESCRIPTION
**Pull request by abdulmuizz0903**

### CODE REVIEW

**Comment by abdulmuizz0903** [comment]
I added Arabic language to the app. It will expands the apps audience to Middle East.

File: app/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<resources>
+    <string name="app_name">Food Expiration Dates</string>
+    <string name="app_name_short">Food Expirations</string>
```
Comment (lorenzovngl) [inline app/src/main/res/values-ar/strings.xml:3]: Could you translate also these strings?

For the string `app_name`, please add the corresponding translation, and for the string `app_name_short`, if there is a short way to translate (as in English), add it, but if there's not and the translation is equal to `app_name`, consider to link the two string as follows:
```
<string name="app_name_short">@string/app_name</string>
```

**Review by lorenzovngl — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ar/strings.xml (hunk 1 above)
Comment (abdulmuizz0903) [reply app/src/main/res/values-ar/strings.xml]: Done. Please take a look in the updates

**Comment by abdulmuizz0903** [comment]
Requested Changes have been made😊.

**Review by lorenzovngl — APPROVED** [review]

**Comment by lorenzovngl** [comment]
Fantastic! I will notice you when there will be new string to translate.
Thank you!

**Pull request merged**